packages/shared/src/components/MainLayout.tsx

```diff
@@ -235,6 +247,16 @@ export default function MainLayout({
           onRequestClose={() => setIsIncompleteRegistration(false)}
         />
       )}
+      {user && !user.infoConfirmed && (
+        <StyledModal isOpen onRequestClose={() => logout()}>
+          <SocialRegistrationForm
+            provider=""
```

Comment:
Maybe just best to make it optional then?

Reply:
Totally agree with that.